src/main/java/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/MagicSugar.java

```diff
@@ -23,6 +26,11 @@ public MagicSugar(Category category, SlimefunItemStack item, RecipeType recipeTy
 	public ItemInteractionHandler getItemHandler() {
 		return (e, p, item) -> {
 			if (isItem(item)) {
+				if(e.getClickedBlock() != null) {
+					String target = BlockStorage.checkID(e.getClickedBlock());
+					if ("ANCIENT_PEDESTAL".equals(target) || "ANCIENT_ALTAR".equals(target))
+						return false;
```

Comment:
I am also thinking whether a Material comparison for ENCHANTMENT_TABLE and DISPENSER may not be a better idea.
It would definitely be better performance-wise.
Also please add a comment to this code to explain why it is there.

Reply:
I was thinking that too but added the check so that people don’t accidentally open inventories when trying to use the sugar. Tbh it would work either way.